src/048/p048.clj

```diff
@@ -0,0 +1,22 @@
+(defn series-step [x]
+  (reduce * (repeat x (bigint x))))
+
+(defn only-last-10-digits [x]
+  (->> x
+       str
+       vec
+       (take-last 10)
+       (apply str)
+       bigint))
+
+(defn sum-series [x] 
+  (transduce 
+    (map (comp only-last-10-digits
+               series-step))
+    +
+    (range 1 (inc x))))
+
+(defn solve [x]
+  (->> x sum-series only-last-10-digits str println))
+
+(solve 1000)
```

Comment:
This one is missing a last line.

Reply:
this specific case _needs_ to be printed with println as a string, since Clojure's "normal" ints can't really take the value, only bigints, and when printed with `prn` (which is actually a pretty-print) has a trailing 'N'